src/Screen/Screen.php

```diff
@@ -236,7 +236,11 @@ private function bind(int $key, ReflectionParameter $parameter, array $httpQuery
         $object = resolve($class);
 
         if ($original !== null && is_a($object, UrlRoutable::class)) {
-            return $object->resolveRouteBinding($original);
+            $object_to_return = $object->resolveRouteBinding($original);
```

Comment:
Variables are written in the style `$objectToReturn`.  By the way, I don't think such a name is appropriate.

Reply:
i agree to change the name of the var,

  dint know var can only be written in camelcase.

Reply:
what about, $boundObject ?   as it make more sense

Reply:
What do you think about changing the early exit from the function so that it would be like this:

```php
$object = resolve($class);

if ($original === null || !is_a($object, UrlRoutable::class)) {
    return $object;
}

return tap($object->resolveRouteBinding($original), function ($object) use ($parameter) {
    Route::current()->setParameter($parameter->getName(), $object);
});
```